python/llm/example/GPU/HuggingFace/LLM/glm-edge/README.md

```diff
@@ -0,0 +1,147 @@
+# GLM-Edge
+In this directory, you will find examples on how you could apply IPEX-LLM INT4 optimizations on GLM-Edge models on [Intel GPUs](../../../README.md). For illustration purposes, we utilize the [THUDM/glm-edge-1.5b-chat](https://hf-mirror.com/THUDM/glm-edge-1.5b-chat) and [THUDM/glm-edge-4b-chat](https://hf-mirror.com/THUDM/glm-edge-4b-chat) as reference GLM-Edge models.
```

Comment:
Please use the huggingface link  (https://huggingface.co/THUDM/glm-edge-1.5b-chat) instead.